### PR TITLE
Added new branding material #4598

### PR DESCRIPTION
**Pull request by gmosx**

### CODE REVIEW

**Comment by ktsaou** [comment]
@paulfantom 	I am not familiar with `git-lfs`. These files are the netdata icons on the dashboard, so they are needed to be cloned together with the repo, or the dashboard will not work.

Is `git-lfs` suitable for this use? Will this introduce any dependencies or exotic procedures for the repo clones?

**Comment by gmosx** [comment]
AFAICS, it will introduce at least an additional dependency on Github (will store the binary files).

**Comment by paulfantom** [comment]
Yeah, so here we get to a place where our update mechanism collides with git best practices and causes us to have bloated repository (already takes 26.86MB in compressed form and 43M in uncompressed one). Basically if you put something in git repo it stays there forever, it is ok if those files are in plaintext since git can do some differential magic and compression on them. Where it fails is binary files. As you cannot do diffs on those, every small change will result in two blobs and greatly increase repo size.
GitHub guys saw this with many repos and created a git plugin called `git-lfs` which allows people to create a form of symlinks pointing to binary files stored in "cloud" (github servers, but others can also be used).

Now to the problem with updater: git-lfs needs to be first installed to be operational.

TL;DR:
- Best practices say that binary files shouldn't be stored in git repo
- Putting binary files in git repo greatly increases repo size
- Our updater needs them there

**Comment by gmosx** [comment]
> (github servers, but others can also be used).

What happens to our binary files if (for any reason) these Github servers become non-operational.

A (slightly) related question: AFAIU, `git rm --cached` removes files from the repo (including the history), is that right?

**Comment by paulfantom** [comment]
From `man git-rm`:

>        --cached
>           Use this option to unstage and remove paths only from the index. Working tree files, whether modified or not, will be left alone.

So no, it does not touch history.

**Comment by ktsaou** [comment]
I understand that the only way to use best practices is to finish packaging.
So, let's merge this and discuss the git-lfs issue and packaging during our meetup.

@gmosx 	please solve the problem with icons refreshing during URL hash updates, to merge this.

**Comment by gmosx** [comment]
> @gmosx please solve the problem with icons refreshing during URL hash updates, to merge this.

Already fixed this.
